### ISSUE TITLE
"A bare-bones test example" issue

### ISSUE THREAD

**Issue by amjankowska** [noveoko/python_1_day_course#2]
The function
```
def skip_every_other_letter(string):
    limited = ""
    keep = range(0,len(string),2)
    for i in keep:
        limited+=string[i]
    return limited
```
doesn't actually pass the test (as you said). You need to change the range to range(1,len(string),2).

**Comment by noveoko** [comment]
The function is supposed to return 'every other char' in a string:
skip_every_other_letter('10101010101010')
Out[4]: '1111111'
OR
skip_every_other_letter('01010101010101')
Out[5]: '0000000'

**Comment by amjankowska** [comment]
yeah, so testing it with the word doe is supposed to give 'de', but somehow it passes the test checking if this string == 'o'

**Comment by amjankowska** [comment]
yeah, so for the word "doe" it returns "de", but in your tutorial you check whether the returned string == "o" and (as you claim) it passes the test. it actually doesn't

**Comment by noveoko** [comment]
I think in this case the test needs to be rewritten. I will put together a few new samples today. Thank you. :)
